.github/workflows/canary-integration-test.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         sed -i 's/<OSD-IDs>/1/' deploy/examples/osd-purge.yaml
         # the CI must force the deletion since we use replica 1 on 2 OSDs
         sed -i 's/false/true/' deploy/examples/osd-purge.yaml
-        sed -i 's|rook/ceph:master|rook/ceph:local-build|' deploy/examples/osd-purge.yaml
+        sed -i 's|rook/ceph:v1.8.1|rook/ceph:local-build|' deploy/examples/osd-purge.yaml
```

Comment:
Broader discussion for us @leseb @travisn :  the CI `canary` test was failing until I made this change. Is it really necessary to sed replace `rook/ceph:<version>` instead of using a replace string that wouldn't need to be modified for each new release? Why don't we use `s|rook/ceph:.*|rook/ceph:local-build|' here?

Reply:
Agreed, let's use the wildcard. I don't see why that wouldn't work.